apps/ccf-eui/src/app/app.component.ts

```diff
@@ -335,4 +291,37 @@ export class AppComponent implements OnInit {
   asMutable<T>(value: Immutable<T>): T {
     return value as T;
   }
+
+  readonly currentOrganInput = model('');
+  readonly organs = model<string[]>([]);
+  readonly filteredOrgans = computed(() => {
+    const currentOrganInput = this.currentOrganInput().toLowerCase();
+    return currentOrganInput
+      ? ALL_ORGANS.filter((organ) => organ.name.toLowerCase().includes(currentOrganInput))
+      : ALL_ORGANS.slice();
+  });
+  readonly searchTextField = computed(() => this.organs().length.toString() + ' Organs Visible');
+
+  remove(organ: string): void {
+    this.organs.update((organs) => {
+      const index = organs.indexOf(organ);
+      if (index < 0) {
+        return organs;
+      }
+
+      organs.splice(index, 1);
```

Comment:
This will modify the original array. Judging be the next line I believe your intention was something like:
```ts
const copy = [...organs];
copy.splice(index, 1);
return copy;
```